packages/design-system-react/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
Added `.eslintrc.js` to support browser environment in React package

This ESLint configuration extends the root config but adds browser-specific settings. The root ESLint config is Node.js-focused, so this configuration:

- Enables browser environment
- Disables Node.js environment
- Turns off `no-restricted-globals` rule to allow browser globals
- Adds TypeScript configuration for .ts/.tsx files